include/picongpu/plugins/PluginController.hpp

```diff
@@ -183,25 +188,25 @@ class PluginController : public ILightweightPlugin
 #endif
     >;
 
-    typedef AllCombinations<
-        bmpl::vector<VectorAllSpecies, UnspecializedSpeciesPlugins>
-    >::type CombinedUnspecializedSpeciesPlugins;
-
-    typedef bmpl::transform<
-        CombinedUnspecializedSpeciesPlugins,
-        ApplyDataToPlugin<bmpl::_1>
-    >::type SpeciesPlugins;
-
-
-    /* create sequence with all plugins*/
-    typedef MakeSeq<
+    using SpeciesPlugins = typename MakeSeqFromNestedSeq<
```

Comment:
removed this section now to use `SpeciesEligibleForSolver` directly with no noticable CT impact.

One reason for the overhead could also be the `MakeSeqFromNestedSeq`.